Assets/StreamingAssets/Quests/A0C00Y16.txt

```diff
@@ -292,9 +294,11 @@ _yes_ task:
 
 _givereward_ task:
 	when _yes_ and _S.18_ 
+	start timer _delay_ -- otherwise gold message is displayed before gratitude message, which seems erroneous
```

Comment:
Comments can only be placed at the start of a newline. The parser uses string.StartsWith("-") and eliminates the whole line when a - character is present.

Also need to be careful to put a full empty line below message blocks when adding a comment. The empty line is what terminates the message block, and comments will be incorporated if grouped with the other text lines in block.

Reply:
I've updated the quest file according to your comments. I have a feeling I bungled things with respect to updating the branch/PR -- please let me know if so.